rabbitmq/publisher.go

```diff
@@ -59,46 +61,46 @@ func checkArgs(args ArgsRabbitMqPublisher) error {
 	}
 
 	if args.Config.EventsExchange.Name == "" {
-		return ErrInvalidRabbitMqExchangeName
+		return fmt.Errorf("%w for %s", ErrInvalidRabbitMqExchangeName, args.Config.EventsExchange.Name)
```

Comment:
```suggestion
		return fmt.Errorf("%w for EventsExchange", ErrInvalidRabbitMqExchangeName)
```
as name is empty anyway. similar for all, including Type

Reply:
right, changed for all